test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -117,14 +113,15 @@ public void VerifyImages(ImageDescriptor imageDescriptor)
             {
                 CreateTestAppWithSdkImage(imageDescriptor, appSdkImage);
 
-                if (!imageDescriptor.IsArm)
+                if (!string.IsNullOrEmpty(imageDescriptor.SdkOsVariant))
                 {
                     VerifySdkImage_RunApp(imageDescriptor, appSdkImage);
                 }
 
                 VerifyRuntimeImage_FrameworkDependentApp(imageDescriptor, appSdkImage);
 
-                if (DockerHelper.IsLinuxContainerModeEnabled)
+                if (DockerHelper.IsLinuxContainerModeEnabled
+                    && !string.Equals(imageDescriptor.OsVariant, "alpine", StringComparison.OrdinalIgnoreCase))
```

Comment:
Consider moving `string.Equals(imageDescriptor.OsVariant, "alpine", StringComparison.OrdinalIgnoreCase)` to a separate method -

```C#
        private static bool IsAlpine(ImageDescriptor imageDescriptor)
        {
            return string.Equals(imageDescriptor.OsVariant, "alpine", StringComparison.OrdinalIgnoreCase)) 

        }
```

Reply:
Fixed as suggested